json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -36,6 +36,18 @@ protected function result() {
 
 		return array( 'scheduled' => Jetpack_Sync_Actions::schedule_full_sync( $modules ) );
 	}
+
+	protected function validate_queue( $query ) {
```

Comment:
I like this. We should probably get rid of the extra newline at the top of the function block though.